doc/source/examples/networktheory/CPW media example.ipynb

```diff
@@ -0,0 +1,271 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# CPW media example\n",
+    "\n",
+    "Coplanar waveguide (CPW) is a planar transmission line were the return conductor is a plane located on both side of a strip conductor on the same layer. The characteristic impedance depends of the strip width and the gap with the return plane, giving two degree of freedom to designer.\n",
+    "\n",
+    "A great advantage of CPW is the convenience to perform shunt or serie connexions with components, because both the signal and the return path are on the same side, thus no via are required. On the other hand, the larges return planes consume a lot of surface on the circuit.\n",
+    "\n",
+    "Coplanar waveguide has quasi-TEM mode and less dispertion than microstrip. However, to avoid effect of slotline-mode, the planes on both side of the strip have to be connected together trough bond-wires. Another solution is to add a conductor plane on the backside of substrate and connect all planes together with via fences. But should the height of the sustrate be small and the gap with coplanar planes be big, then the line would behave like a microstrip line instead.\n",
+    "\n",
+    "![isometric view of coplanar waveguide structure without conductor backing](coplanar_waveguide_structure.svg \"coplanar waveguide structure without conductor backing\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import skrf as rf\n",
+    "from skrf.calibration import IEEEP370_SE_NZC_2xThru\n",
+    "from skrf.media import CPW\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "rf.stylely()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Measurement of two CPWG lines with different lengths\n",
+    "\n",
+    "The measurement where performed the 21th March 2017 on a Anritsu MS46524B 20GHz Vector Network Analyser. The setup is a linear frequency sweep from 1MHz to 10GHz with 10'000 points. Output power is 0dBm, IF bandwidth is 1kHz and neither averaging nor smoothing are used.\n",
```

Comment:
typo: "were".

There are probably a lot more, but I'm not going to spell check the rest. It should be checked with a spell checker.

Reply:
Thank you a lot ! I will try with grammarly.